pkg/minikube/reason/known_issues.go

```diff
@@ -874,6 +874,15 @@ var internetIssues = []match{
 }
 
 var guestIssues = []match{
+	{
+		Kind: Kind{
+			ID:       "GUEST_PROVISION_NOSPACE_DOCKER",
+			ExitCode: ExInsufficientStorage,
+			Advice:   "Ensure you have at least 20GB of free disk space accessible to Docker Desktop",
+		},
+		// https://github.com/kubernetes/minikube/issues/11452
+		Regexp: re(`docker: failed to register layer: ApplyLayer exit status 1`),
```

Comment:
This message from docker can indicate a lot of possible issues, not only insufficient disk space
See for example https://bugzilla.redhat.com/show_bug.cgi?id=1311758

Reply:
that a good point !

how about 

```
	Kind: Kind{
			ID:       "GUEST_PROVISION_DOCKER_INTERNAL",
			ExitCode: ExInternal,
			Advice:   "Ensure your Docker installation is healthy",
```